src/docs/sdk/event-payloads/stacktrace.mdx

```diff
@@ -24,6 +24,12 @@ follow this rule of thumb:
 : **Required**. A non-empty list of stack frames (see below). The list is
   ordered from caller to callee, or oldest to youngest. The last frame is the
   one creating the exception.
```

Comment:
@MimiDumpling the paragraph you're adding is replicating this description here, it doesn't look right to have both.

Reply:
@rhcarvalho Yes, good point. We discussed adding the Note to the #attributes section of this page. I looked for other places for the Note, but there's no good fit. Perhaps we don't need the Note?